internal/commands/extension_inspect.go

```diff
@@ -24,18 +28,39 @@ func ExtensionInspect(logger logging.Logger, cfg config.Config, client PackClien
 			extensionName := args[0]
 			registry := flags.Registry
 			if registry == "" {
-				// fix registry for extension
+				registry = cfg.DefaultRegistryName
 			}
 
 			return extensionInspect(logger, extensionName, registry, flags, cfg, client)
 		}),
 	}
-	// flags will be added here
+	cmd.Flags().IntVarP(&flags.Depth, "depth", "d", -1, "Max depth to display for Detection Order.\nOmission of this flag or values < 0 will display the entire tree.")
```

Comment:
Ah, here is one more that could be removed (since there is no order)

Reply:
Yes, making all the changes, today everything will be done ☺️